app/models/guide.rb

```diff
@@ -0,0 +1,8 @@
+class Guide < ActiveRecord::Base
+  belongs_to :user
+
+  #validates :user_id, presence: true
```

Comment:
This should not be commented out; did while trying to debug the test suite. Already fixed locally.